apps/server/src/shared/infra/identity-management/keycloak/controller/console/keycloak-management.console.ts

```diff
@@ -115,13 +119,19 @@ export class KeycloakConsole {
 		delay: number
 	): Promise<T> {
 		let repetitions = 0;
+		let error = new Error();
 		while (repetitions < count) {
 			repetitions += 1;
 			try {
 				// eslint-disable-next-line no-await-in-loop
 				return await command();
 			} catch (err) {
-				this.console.info(JSON.stringify(err));
+				if (err instanceof Error) {
+					error = err;
+				} else {
+					this.logger.error(err);
```

Comment:
You still need to set the error variable. Otherwise the error will be thrown as it was initialized in line 122.

This branch is covered by tests, but it is only checked with `toThrow()` it is not checked, that the correct error is thrown.

Reply:
A test was written for it, see line 98.